lib/security/security-spec.js

```diff
@@ -0,0 +1,70 @@
+var assert = require('assert');
+var sampleUserProfileData = require('../../test/fixtures/sampleUserProfile.json');
+var aes = require('./aes.js');
+var sha256 = require('./sha256.js');
+var sampleSecurityeData = require('../../test/fixtures/sampleSecurityData.json');
+
+describe("Test Data Encyption, Hashing and Decryption", function() {
+  it('should error when you try to hash a null value', function(done) {
```

Comment:
When testing with Mocha and promises, you can leave out the `done` callback, and just return the promise itself. If the promise resolves, then the test passes. If the promise chain fails (like if an assertion throws), then the test fails.